pinot-common/src/test/java/org/apache/pinot/common/metrics/prometheus/dropwizard/DropwizardBrokerPrometheusMetricsTest.java

```diff
@@ -28,7 +31,7 @@
 /**
  * Disabling tests as Pinot currently uses Yammer and these tests fail for for {@link DropwizardMetricsFactory}
  */
-@Test(enabled = false)
+@Test(enabled = false) // enabled=false on class level doesn't seem to work in intellij
```

Comment:
I'm not sure if we should modify our tests just because intellij doesn't honor some standard testng property.

Reply:
If we don't then running project test suite in IJ will always fail on these.